components/CreateCandidateForm/CreateCandidateForm.tsx

```diff
@@ -18,9 +20,11 @@ export function CreateCandidateForm(): JSX.Element {
         formData.append('firstName', firstName);
         formData.append('lastName', lastName);
         formData.append('email', email);
-
+        const sessionId = getSessionIdFromCookie(document.cookie);
```

Comment:
Does this definitely work?

elsewhere `getSessionIdFromCookie` is passed a single cookie to work with, but here we are passing it all the cookies!

I guess this _probably_ works as long as we don't have 2 cookies with a `sessionId` value?? Probably happens to be true for now?

Reply:
That true, I think I should parse the whole selection of cookies into the the finder and do the search in there,

Reply:
I am now pulling the specific cookie from a new helper